source/CurieBleSerial.cpp

```diff
@@ -0,0 +1,378 @@
+/*
```

Comment:
Following up - removing these files totally prevents the proper metadata from being generated, which in turns prevents CurieBleSerial and DfRobotBleSerial from being accessed. I've added .cpp files with the minimal needed code to force correct compilation.